ew/cmd/cmds/cmdcmds.py

```diff
@@ -2384,47 +2384,7 @@ async def yoslimernalia(cmd: EwCmd):
     ewstats.increment_stat(id_server=cmd.guild.id, id_user=cmd.message.author.id, metric=ewcfg.stat_festivity)
     await fe_utils.send_message(cmd.client, cmd.message.channel, '@everyone Yo, Slimernalia!', filter_everyone=False)
 
-
-async def slimernaliastage(cmd: EwCmd):
-    """ Returns the current slimernalia event stage. """
-    return await fe_utils.send_response(f"Slimernalia is currently at Stage {ewcfg.slimernalia_stage}.", cmd)
-
-
 # Admin commands
-async def announceslimernaliastage(cmd: EwCmd):
-    """ Announces the patch notes for the current/past slimernalia event stages. """
-    if not cmd.message.author.guild_permissions.administrator:
-        return
-    # Unpack tokens, if 'all' is anywhere after the main commad print everything up to where we are currently at
-    # e.g. !announcefestivestage when you're at stage 4 will just announce stage 4
-    # ! announcefestivestage all when you're at stage 4 will announce stages 1, 2, 3 & 4
-    print_all = False
-    if "all" in cmd.tokens:
-        print_all = True
-
-    await fe_utils.announce_slimernalia_stage_increase(cmd.client, cmd.guild, print_all)
```

Comment:
why are we removing this completely? isn't it all sectioned off using the slimernalia_active bit?

Reply:
I wanted to remove slimernalia stage increases, it was a one time thing where i wanted to "stage" new parts of the game so people can enjoy them without being overwhelmed. If someone else wants to do slimernalia stages next year for any purpose then I don't see a reason to remove it.